test/functional/tests/fault_injection/test_fault_injection_core_in_raid.py

```diff
@@ -0,0 +1,68 @@
+
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3
+#
+
+import pytest
+from api.cas import casadm, cli_messages
+from api.cas.cache_config import CacheMode
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet
+from storage_devices.raid import Raid, RaidConfiguration, MetadataVariant, Level
+from test_utils.size import Size, Unit
+from api.cas.cli_messages import (
+    partition_not_suitable_for_array,
+    device_or_resource_busy,
+    check_stderr_msg_any,
```

Comment:
"check_stderr_msg_any" and "check_stderr_msg_all" definitions are missing? Maybe changes in cli_messages file were not committed?

Reply:
Sry, my bad. Fixed